pkg/config/validation/validation.go

```diff
@@ -3910,3 +3915,10 @@ func validateWasmPluginMatch(selectors []*extensions.WasmPlugin_TrafficSelector)
 	}
 	return nil
 }
+
+func validatePartialWildCard(host string, portProto protocol.Instance) error {
+	if strings.Contains(host, "*") && !strings.HasPrefix(host, "*.") {
```

Comment:
what about just plain `"*"`

Reply:
`*.my.api.myservice.com` is valid.

Reply:
This is not completely correct, we also allow `ns/host`